lib/helper-functions.unit.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env jest */
+
+const helperFunctions = require('../lib/helper-functions')
+
+describe('componentNameToMacroName', () => {
+  test('transfoms a single word component name', () => {
+    var macroName = helperFunctions.componentNameToMacroName('button')
+
+    expect(macroName).toBe('govukButton')
+  })
+
+  test('transfoms a multi-word component name', () => {
+    var macroName = helperFunctions.componentNameToMacroName('character-count')
+
+    expect(macroName).toBe('govukCharacterCount')
+  })
+})
+
+describe('componentNameToJavaScriptModuleName', () => {
+  test('transfoms a single word component name', () => {
+    var macroName = helperFunctions.componentNameToJavaScriptModuleName('button')
+
+    expect(macroName).toBe('GOVUKFrontend.Button')
```

Comment:
These aren't macro names any more, so possibly need a different variable name? (Same for the function below)

```suggestion
    var moduleName = helperFunctions.componentNameToJavaScriptModuleName('button')

    expect(moduleName).toBe('GOVUKFrontend.Button')
```

(Alternatively, you could get rid of the variables entirely?)

```suggestion
    expect(helperFunctions.componentNameToJavaScriptModuleName('button'))
      .toBe('GOVUKFrontend.Button')
```